crates/lib/src/spec.rs

```diff
@@ -227,6 +227,10 @@ pub struct BootEntry {
     /// This is true if (relative to the booted system) this is a possible target for a soft reboot
     #[serde(default)]
     pub soft_reboot_capable: bool,
+    /// Whether this deployment is locked from automatic finalization on shutdown
+    #[serde(default)]
+    #[serde(skip_serializing_if = "is_false")]
```

Comment:
I would actually lean towards emitting this always out of consistency, even though yes...it would require changing all the test fixtures (again).